src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -70,9 +70,9 @@ public PersistentDataStore() {
         Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
         User user = new User(uuid, userName, passwordHash, creationTime);
         String aboutMe = (String) entity.getProperty("about_me");
-        boolean isAdmin = (boolean) entity.getProperty("is_admin");
+        Boolean isAdmin = (Boolean) entity.getProperty("is_admin");
         if(aboutMe != null) user.setAboutMe(aboutMe);
-        if(isAdmin == true) user.setIsAdmin(isAdmin);
+        if(isAdmin != null && isAdmin == true) user.setIsAdmin(isAdmin);
```

Comment:
Shouldn't we set is admin whether it's T/F?

Reply:
isAdmin is set to false initially in the constructor.